stock_average_daily_sale/tests/test_average_daily_sale.py

```diff
@@ -188,10 +190,20 @@ def test_average_sale_profile_a(self):
 
     def test_view_refreshed(self):
         self._refresh()
-        with self.assertNoLogs(
+        with self.assertLogs(
             "odoo.addons.stock_average_daily_sale.models.stock_average_daily_sale",
             level="DEBUG",
-        ):
+        ) as cm:
+            logging.getLogger(
```

Comment:
I'm not sure to understand this

Reply:
I added a comment explaining why I changed it and why in that way.